src/dialogs/settingsdialog.cpp

```diff
@@ -164,10 +166,10 @@ void SettingsDialog::readSettings() {
             settings.value("notifyAllExternalModifications").toBool());
     ui->noteSaveIntervalTime->setValue(
             settings.value("noteSaveIntervalTime").toInt());
-    ui->toolbarIconSizeSpinBox->setValue(
```

Comment:
Sorry I just used it for test case to understand better how it works and forgot to reverse it